### PR TITLE
Init skeleton

### PR DESCRIPTION
**Pull request by galta95**
closes #7

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ["@commitlint/config-conventional"],
+  rules: {
+    "scope-enum": [2, "always", ["osmInterface", "dependencies", "configurations"]],
```
Comment (syncush) [inline commitlint.config.js]: osmInterface is not a valid scope, 
think of the type of elements osm exposes: node, way, relation , changeset

File: package.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "node-osm-elements",
```
Comment (syncush) [inline package.json]: missing @map-colonies

**Review by syncush — CHANGES_REQUESTED** [review]

File: commitlint.config.js (hunk 1 above)
Comment (galta95) [reply commitlint.config.js]: done

File: package.json (hunk 2 above)
Comment (galta95) [reply package.json]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**